app/models/crime_application.rb

```diff
@@ -25,4 +26,15 @@ def set_overall_offence_class
       offences: submitted_application['case_details']['offences']
     ).offence_class
   end
+
+  # Replicate the 'hearing_court_name' into 'first_court_hearing_name' to maintain
+  # data consistency for reporting and consuming services
+  def copy_first_court_hearing_name
+    return if submitted_application.blank?
+    return if submitted_application.dig('case_details', 'first_court_hearing_name').present?
+    return if submitted_application.dig('case_details', 'is_first_court_hearing') == 'no'
```

Comment:
Wonder if we could use the value object YesNoAnswer::No here rather than having the 'no' string.